### ISSUE TITLE
Sort

### ISSUE THREAD

**Issue by sfxworks** [PhantomAppDevelopment/firebase-as3#20]
Do all of these methods work?
https://firebase.google.com/docs/database/rest/retrieve-data

I am trying to order my data alphabetically. After I turn my data into an Object I can't turn it into an array and sort it.

```
var sorted:Array = e.data //JSON.parse result
sorted.sortOn("1");
```

I saw that REST isn't able to keep list items at order either. I want to sort all the items I get from a JSON node alphanumerically.

**Comment by agentphantom** [comment]
You are correct, Firebase always returns the data scrambled and you are required to sort it in client side.

Please take a look at this [function](https://github.com/PhantomAppDevelopment/pizza-app/blob/master/src/galleryScreens/GalleryScreen.as#L163), it shows you how you can recursively convert the JSON into an Array, sort it and pass it to a data provider.
